runtime/doc/ui.txt

```diff
@@ -55,8 +55,8 @@ with these (optional) keys:
 	`stdin_fd`		Read buffer from `fd` as if it was a stdin pipe
 				This option can only used by |--embed| ui,
 				see |ui-startup-stdin|.
-
-
+	`term_ttyin`		Tells if `stdin` is a `tty` or not.
```

Comment:
for parallel form with `stdin_fd` this could be named `stdin_tty` , `stdout_tty`